### ISSUE TITLE
seperator strings cannot be "any list of unicode characters"

### ISSUE THREAD

**Issue by cmaeder** [tillmo/DOL#86]
```
Separators       = 'separators' , String , String ;
String           = ($<$ \rm any list of unicode characters $>$) ;
```

String would match the remaining content (and fail the read a second non-empty String). 

Change it to "any non-empty list of printable unicode characters that are no unicode spaces"
(https://github.com/spechub/Hets/issues/1366)

**Comment by gnn** [comment]
Please note that I did not close the issue as I would like somebody to have a short look at the change and approve it by closing the issue or give a short comment so I can add another patch.

**Comment by gnn** [comment]
The fix to this issue has been amended with b73d0d6dc6bb072520b10cdd95b5f7ab661cca88 for readability and with 84b2e8976ca6d87a56c233a8cfe2de1777a3d394 to finally disallow '#' for safety reasons but allowing other funky characters (those from gen-delims) in addition to "ipchar"s.
Closing this issue as @cmaeder and me agreed that this should suffice.
